ui/src/types/labels.ts

```diff
@@ -1,8 +1,4 @@
-import {Label as GenLabel} from 'src/client'
-
-export interface Label extends GenLabel {
-  properties: LabelProperties
-}
+export {Label} from 'src/client'
```

Comment:
I don't understand this.  should be as it was before. 

The client Label doesn't have properties, we are adding required properties. of type LabelProperties.

Reply:
The old client library did have LabelProperties that were set differently for the `ILabel`:

https://github.com/influxdata/influxdb-client-js/blob/master/src/types/labels.ts#L13

This had originally been set to ensure that we had type stability. However, now that the labels have been refactored it is no longer necessary